gcm_filters/kernels.py

```diff
@@ -113,6 +115,8 @@ class IrregularCartesianLaplacianWithLandMask(BaseLaplacian):
     dxs: ArrayType
     dys: ArrayType
     area: ArrayType
+    kappa_w: ArrayType
+    kappa_s: ArrayType
```

Comment:
Thanks for this addition! It would be great to include a check that the user does not input kappa > 1, which would cause the filter to blow up (as discussed by @iangrooms and @sdbachman on slack).

Reply:
Ok, I will add this check.